src/rdmnet/broker/broker_uid_manager.h

```diff
@@ -77,6 +85,8 @@ class BrokerUidManager
   // The next dynamic RDM Device ID that will be assigned
   uint32_t next_device_id_{1};
   size_t   max_uid_capacity_{kDefaultMaxUidCapacity};
+  // Protects the members of BrokerUidManager
+  mutable std::unique_ptr<etcpal::RwLock> uid_manager_lock_;
```

Comment:
We already know we're in a UidManager instance, so this can just be called `lock_`